src/web/modules/transactions/list.njk

```diff
@@ -140,11 +145,11 @@
       {% if not(set._links.next_page) %}
         disabled
       {% elseif filters.gateway_payout_id %}
-        href="/transactions?page={{ set.page + 1 }}&account={{ accountId }}&gateway_payout_id={{ filters.gateway_payout_id }}"
+        href="/transactions?page={{ set.page + 1 }}&amp;account={{ accountId }}&amp;gateway_payout_id={{ filters.gateway_payout_id }}"
       {% elseif filters.agreement_id %}
-        href="/transactions?page={{ set.page + 1 }}&account={{ accountId }}&agreement_id={{ filters.agreement_id }}"
+        href="/transactions?page={{ set.page + 1 }}&amp;account={{ accountId }}&amp;agreement_id={{ filters.agreement_id }}"
       {% else %}
-        href="/transactions?page={{ set.page + 1 }}&status={{ selectedStatus }}{{ linkQueryParams }}"
+        href="/transactions?page={{ set.page + 1 }}&amp;status={{ selectedStatus }}{{ linkQueryParams }}"
```

Comment:
We don't encode these elsewhere (eg on the agreements page nav links). Is this causing a problem?

Reply:
Other than that it annoys me, no, not really. I just thought I’d fix it while I was editing that page.

Reply:
OK - we should probably be consistent across the app if we do this, there are a a few dotted around the templates

Reply:
I’ll consistently fix them when I have a consistent few minutes.